packages/@aws-cdk/aws-iot-actions-alpha/README.md

```diff
@@ -163,6 +163,24 @@ const topicRule = new iot.TopicRule(this, 'TopicRule', {
 });
 ```
 
+## Start Step Functions State Machine
+
+The code snippet below creates an AWS IoT Rule that starts a Step Functions State Machine 
+when it is triggered.
+
+```ts
+const stateMachine = new stepfunctions.StateMachine(this, 'SM', {
+  definitionBody: stepfunctions.DefinitionBody.fromChainable(new stepfunctions.Wait(this, 'Hello', { time: stepfunctions.WaitTime.duration(Duration.seconds(10)) })),
+});
+
+new iot.TopicRule(this, 'TopicRule', {
+  sql: iot.IotSql.fromStringAsVer20160323("SELECT * FROM 'device/+/data'"),
+  actions: [
+    new actions.StepFunctionsStartStateMachineAction(stateMachine),
```

Comment:
Hey, `StepFunctionsStartStateMachineAction` could be renamed to `StepFunctionsStateMachineAction`.